contrib/chibi-genstatic-helper.cmake

```diff
@@ -16,7 +16,7 @@ if(NOT OUT)
 endif()
 
 execute_process(
-    COMMAND ${EXEC} ${GENSTATIC} --no-inline
+    COMMAND ${EXEC} -q -I ./lib ${GENSTATIC} --no-inline
```

Comment:
The working directory for executing this command is already set from the top-level `CMakeLists.txt`, so I don't think we need it here. Interestingly, I don't even know why I put in the `-I ./lib` flag. At some point I experimented with separate `default_module_path`s for build and installation configurations (which was a bad idea, and I reverted it) - this might even be a leftover from this experiment.

Reply:
So it's not required any longer (even for POSIX systems)? Reducing diffs is generally good idea.

Reply:
Yep, the flags were superfluous - I removed them.